pages/tutorials/index.html

```diff
@@ -107,7 +117,7 @@ <h1 class="card-title">Users' Guides.</h1>
         <div class="card-footer">
           <a href="../cython/index.html" class="btn btn-primary">Python </a>
```

Comment:
This link is going to be `../documentation/index.html#python` and the Matlab one similarly

Reply:
Alternatively, making it relative to the site root would probably be better: `/documentation/index.html`